### ISSUE TITLE
There was a problem while training.

### ISSUE THREAD

**Issue by emmawatsonll** [experiencor/basic-yolo-keras#323]
![default](https://user-images.githubusercontent.com/37065176/42290754-3f91106c-8002-11e8-81a4-2421b11511f3.jpg)

Why is this happening?
CRY = 0.00 , ANGRY = 0.00 ... 
Problem with Batch Size?
Also, I wonder why it is early stop.

Help, please!

**Comment by letilessa** [comment]
Hi Emma, check the parameters in your early stop callback. Your training is being interrupted because the loss is too low and not showing any improvement according to what is defined in the callback.

`early_stop = EarlyStopping(monitor='val_loss', 
                           min_delta=0.001, 
                           patience=3, 
                           mode='min', 
                           verbose=1)`
